Dockerfile

```diff
@@ -2,16 +2,10 @@ FROM node:16-alpine
 
 # RUN yarn set version berry
 # Copying repo resources
-COPY ./package.json ./
-COPY ./tsconfig.json ./
 COPY ./packages ./packages
-COPY ./nodemon.json ./
-COPY ./.eslintrc.js ./
 COPY ./apps ./apps
```

Comment:
I am not sure if this copy command is consistent with the volume mappings in docker-compose.yml.

Reply:
COPY cmd copy asset to image (static binding) build phase.
VOLUME mounting is attaching local file system with running container (Dynamic).

Reply:
I think we shouldn't need to copy if we mount directory as a volume